resolwe/flow/managers/dispatcher.py

```diff
@@ -755,13 +762,7 @@ def process_data_object(data):
                 lambda d=data, p=program: self._data_execute(d, p, executor, verbosity)
             )
 
-        logger.debug(__(
-            "Manager '{}.{}' processing communicate command on '{}' for executor '{}'.",
-            self.__class__.__module__,
-            self.__class__.__name__,
-            state.MANAGER_CONTROL_CHANNEL,
-            executor
-        ))
+        logger.debug(__("Manager processing communicate command triggered by data id {}.", data_id))
```

Comment:
Similar as above, I would change this to `logger.debug(__("Manager processing communicate command triggered by Data with id {}.", data_id))`.